lib/NormalModule.js

```diff
@@ -79,6 +83,28 @@ const ABSOLUTE_PATH_REGEX = /^([a-zA-Z]:\\|\\\\|\/)/;
  * @property {string?} type
  */
 
+/**
+ * @typedef {Object} LoaderContextBase
```

Comment:
You could also write these types in typescript in `declarations/LoaderContext.d.ts` if you want to.